libguard/guard_interface.cpp

```diff
@@ -169,6 +188,9 @@ GuardRecord create(const EntityPath& entityPath, uint32_t eId, uint8_t eType)
         lastPos++;
     }
 
+	// Space left in file before writing new record
```

Comment:
a new record?

Reply:
Yes, before creating new guard record we are checking for the space in guard file.

Reply:
> Space left in file before writing new record

Space left in file before writing new a record

Reply:
Done.